app/views/base/layout.scala

```diff
@@ -116,6 +116,14 @@ object layout:
       div(id := "dasher_app", cls := "dropdown")
     )
 
+  private def anonDasher(using ctx: PageContext) =
+    val prefs = trans.preferences.preferences.txt()
+    div(cls := "dasher")(
+      a(href := s"${routes.Auth.login.url}?referred=${ctx.req.path}", cls := "signin")(trans.signIn.txt()),
+      a(cls  := "toggle anon link", title := prefs, aria.label := prefs, dataIcon := licon.Gear),
```

Comment:
why has this turned into a `a`? It has no `href`

Reply:
not sure, will fix.